.github/workflows/pythonpackage.yml

```diff
@@ -55,3 +56,9 @@ jobs:
     - name: Upload coverage details
       run: |
         codecov
+    - name: Style checks pylint
+      run: |
+        python -m pylint dulwich examples
+    - name: Test unittest
```

Comment:
This is already done on line 52.

Reply:
Sorry I didn't see it, thank you for your reminder.